src/uu/chown/src/chown.rs

```diff
@@ -149,8 +154,10 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 dest_uid = Some(meta.uid());
             }
             Err(e) => {
-                show_error!("failed to get attributes of '{}': {}", file, e);
-                return 1;
+                return Err(USimpleError::new(
+                    1,
+                    format!("failed to get attributes of '{}': {}", file, e),
+                ));
```

Comment:
`fs::metadata` returns an `io::Result`, so I think this is a nice opportunity to use `UIoResult`:
```rust
let meta = fd::metadata(&file)
    .map_err_context(|| format!("failed to get attributes of '{}'"))?;
let dest_gid = Some(meta.gid());
let dest_uid = Some(meta.uid());
```